src/bundle.js

```diff
@@ -7,8 +7,30 @@ var _define = window.define;
 // jstree
 var jstree = require("jstree");
 require("jstree/dist/themes/default/style.css");
-_define("jstree", ["jQuery"], function() { return jstree; });
+_define("jstree", function() { return jstree; });
```

Comment:
jstree has a dependency on jQuery so I wonder if removing this could introduce some weird issues.

Reply:
I tested the metadata manager after removing this and didn't notice anything wrong. I think it's fine because I bundled jquery as well.